js/bybit.js

```diff
@@ -3564,6 +3584,99 @@ module.exports = class bybit extends Exchange {
         return this.parseTrades (result, market, since, limit);
     }
 
+    parseDepositAddress (depositAddress, currency = undefined) {
+        //
+        //     {
+        //         chain_type: 'Arbitrum One',
+        //         address_deposit: '0x83a127952d266A6eA306c40Ac62A4a70668FE3BE',
+        //         tag_deposit: '',
+        //         chain: 'ARBI'
+        //     }
+        //
+        const address = this.safeString (depositAddress, 'address_deposit');
+        const tag = this.safeString (depositAddress, 'tag_deposit');
```

Comment:
If the tag is an empty string, we should set it to `undefined`:

```suggestion
        let tag = this.safeString (depositAddress, 'tag_deposit');
        if (tag === '') {
            tag = undefined;
        }
```